<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDebug</class>
 <widget class="QWidget" name="ConfigureDebug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GDB</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_1">
          <item>
           <widget class="QCheckBox" name="toggle_gdbstub">
            <property name="text">
             <string>Enable GDB Stub</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="gdbport_spinbox">
            <property name="maximum">
             <number>65536</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Logging</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Global Log Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="log_filter_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Regex Log Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="log_regex_filter_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="toggle_console">
          <property name="text">
           <string>Show Log Console (Windows Only)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="open_log_button">
          <property name="text">
           <string>Open Log Location</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>CPU</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QWidget" name="clock_speed_widget" native="true">
        <layout class="QHBoxLayout" name="clock_speed_layout">
         <property name="spacing">
          <number>7</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="clock_speed_combo">
           <item>
            <property name="text">
             <string>Use global clock speed</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Set clock speed:</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="clock_speed_label">
           <property name="text">
            <string>CPU Clock Speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_clock_speed">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;body&gt;Changes the emulated CPU clock frequency.&lt;br&gt;Underclocking can increase performance but may cause the game to freeze.&lt;br&gt;Overclocking may reduce in game lag but also might cause freezes&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>79</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="value">
            <number>25</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="clock_display_label">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="toggle_cpu_jit">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enables the use of the ARM JIT compiler for emulating the 3DS CPUs. Don't disable unless for debugging purposes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Enable CPU JIT</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="toggle_renderer_debug">
        <property name="text">
         <string>Enable debug renderer</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="toggle_dump_command_buffers">
        <property name="text">
         <string>Dump command buffers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
    <item>
      <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
          <string>Miscellaneus</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
            <widget class="QCheckBox" name="delay_start_for_lle_modules">
              <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Introduces a delay to the first ever launched app thread if LLE modules are enabled, to allow them to initialize.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
                <string>Delay app start for LLE module initialization</string>
              </property>
            </widget>
          </item>
        </layout>
      </widget>
    </item>
   <item>
    <widget class="QLabel" name="label_cpu_clock_info">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CPU Clock Speed Information&lt;br/&gt;Underclocking can increase performance but may cause the game to freeze.&lt;br/&gt;Overclocking may reduce in game lag but also might cause freezes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toggle_gdbstub</tabstop>
  <tabstop>gdbport_spinbox</tabstop>
  <tabstop>log_filter_edit</tabstop>
  <tabstop>toggle_console</tabstop>
  <tabstop>open_log_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>toggle_gdbstub</sender>
   <signal>toggled(bool)</signal>
   <receiver>gdbport_spinbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
